### DISCUSSION TITLE
Blocked in Indonesia because of new digital ecosystem law

### DISCUSSION THREAD

**Discussion by TimotiusAnrez** [godotengine/godot-proposals#5039] [General Discussions]
Hi, I am a software developer from Indonesia.

Before everything I would like to apologize if there is any missing information, or assumptions regarding legal ground of godot. As of jully 30 2022, Indonesian ministry of communication and information blocked a lot of digital company that operates in Indonesia from paypal to steam. And this include godot, since godot is open source will there be any contacts with Indonesia ministry of communication regarding this issue or will there be nothing that anyone can do since there is no legal team that can deal with this issue?

**Comment by AlexVSharp** [comment]
_Disclaimer: I'm not part of the Godot team, and I haven't been following the [fall](https://freedomhouse.org/country/indonesia/freedom-net/2021) of Indonesia closely enough to know the ins and outs._

However, I am familiar with the types in charge there, and how they [operate](https://rsf.org/en/country/indonesia).
For confused readers, the op is referring to their [ludicrous move to control data](https://www.reuters.com/technology/indonesia-blocks-yahoo-paypal-gaming-websites-over-licence-breaches-2022-07-30/) which, allegedly, affected Godot as well.

Hopefully someone better versed in Indonesian… law… can prove me wrong, but I think there's a 99% chance that nothing can be done about this at the moment. If they are, paradoxically, going after open-licenced software, then it is a much bigger issue than Godot being inaccessible. :fearful:

**Reply by TimotiusAnrez** [reply]
just an update, Godot has been unblocked as of august 2nd. Tho it cannot be said what makes them (ministry of communication and information) want to block open-licensed software, without thinking about the complication that comes with their decision.
